composer.json

```diff
@@ -7,13 +13,17 @@
 		"christian-riesen/otp": "2.*",
 		"christophwurst/nextcloud_testing": "^0.12",
 		"nextcloud/coding-standard": "^1.0.0",
-		"psalm/phar": "^4.3"
+		"nextcloud/ocp": "dev-stable26",
```

Comment:
let's not install it always because it pollutes the namespaces on dev envs. installation on ci is sufficient